yb-voyager/cmd/common.go

```diff
@@ -218,15 +218,11 @@ func CreateMigrationProjectIfNotExists(dbType string, exportDir string) {
 		}
 	}
 
-	// Put info to metainfo/schema about the source db
-	sourceInfoFile := filepath.Join(projectDirPath, "metainfo", "schema", "source-db-"+dbType)
-	_, err := exec.Command("touch", sourceInfoFile).CombinedOutput()
-	if err != nil {
-		utils.ErrExit("coludn't touch file %q: %v", sourceInfoFile, err)
-	}
+	// Clear any old migration db type and recreating it
+	clearSourceDbType(projectDirPath)
```

Comment:
Can't we keep it where the code for `--start-clean` lives?

Reply:
I have added it with `startClean` also. 
Thought of having this as a extra safety check, like cleaning and recreating it for every migration.